trigger/patch.go

```diff
@@ -203,9 +203,16 @@ func (t *patchTriggers) makeData(sub *event.Subscription) (*commonTemplateData,
 		PastTenseStatus:   t.data.Status,
 		apiModel:          &api,
 		githubState:       message.GithubStatePending,
-		githubContext:     "evergreen",
 		githubDescription: "tasks are running",
 	}
+
+	if t.patch.IsChild() {
+		GitHashLastFour := t.patch.Githash[len(t.patch.Githash)-4:]
+		data.githubContext = fmt.Sprintf("evergreen/%s/%s", t.patch.GithubPatchData.BaseBranch, GitHashLastFour)
```

Comment:
did you decide not to go with the child project's name?

Reply:
I wasn't sure if this exists in the patches collection in the db...

Reply:
We do [set it on the patch](https://github.com/evergreen-ci/evergreen/blob/2f99d8a2ee363999f3d4751e559f0748f408003b/units/patch_intent.go#L710), though the ID might not be what we want these days.